pkg/network/state.go

```diff
@@ -219,8 +224,11 @@ func (ns *networkState) GetDelta(
 	}
 
 	return Delta{
-		Connections: conns,
-		HTTP:        ns.getHTTPDelta(id),
+		BufferedData: BufferedData{
+			Conns:  clientBuffer.Connections(),
```

Comment:
You could use `conns` here since you already call `clientBuffer.Connections()` above.